firmware/arduino/chessboard/chessboard.ino

```diff
@@ -55,11 +55,9 @@ int currPositionX, currPositionY;
 // Maximum position that currPositionX/Y may reach
 int maxPosition;
 
-// Motor directions
-#define UP 0 
-#define DOWN 1
-#define LEFT 0
-#define RIGHT 1 
+// Sets direction of motor to move in the positive or negative direction regardless of axis
```

Comment:
Can you add that right and up are positive, down and left are negative? Something like, "Since the origin (0, 0) is at the bottom left corner, UP and RIGHT correspond to POS_DIR, and DOWN and LEFT correspond to NEG_DIR.